README.md

```diff
@@ -70,17 +70,17 @@ and flags are:
     	Setting for runtime.GOMAXPROCS, <1 doesn't change the default
   -grpc
     	Use GRPC (health check) for load testing
-  -grpc-port int
-    	grpc port (default 8079)
+  -grpc-port string
+      grpc server port. Can take the form of host:port, ip:port or port (default 8079)
```

Comment:
there is some subtlety/manual tweaks on the readme with respect to white space, make sure it matches

also do hand wrap at 80 columns

also can you add the current/future version at line 46 (0.6.7 -> 0.6.8)